uint8 pin
uint8 value
---
uint8 result
